serverless.yml

```diff
@@ -0,0 +1,28 @@
+service: imputation
```

Comment:
Don't think this file should be here - seems to be defining step functions for movement and means and nothing to do with enrichment.